### PR TITLE
New widgets customization components

### PR DESCRIPTION
**Pull request by jberends**
To be compatible with KE-chain-2 release 1 september 2017

- new widget components, leave old inspector components in place
- check jsonschema
- individual widgets should have same behaviour: instantiate from the customization json from the backend and creating them interactively from the code and submit them as customization to the backend.

### CODE REVIEW

**Comment by raduiordache** [comment]
@BennievanderWel Let me know when you want to start working on it

**Comment by raduiordache** [comment]
@BennievanderWel, is it possible to also add a get_customization() function that retrieves the widget_config of an Activity as Customization object?

File: tests/classes.py [hunk 1]
```diff
@@ -26,10 +26,10 @@ def setUp(self):
         if TEST_TOKEN:
             self.client.login(token=TEST_TOKEN)
 
-        self.recorder = Betamax(session=self.client.session)
-        self.recorder.use_cassette(self.cassette_name)
-        self.recorder.start()
+        # self.recorder = Betamax(session=self.client.session)
```
Comment (jberends) [inline tests/classes.py]: I guess this is temporary turned off. Lets make sure this is not in the final commit

File: pykechain/enums.py [hunk 2]
```diff
@@ -69,6 +69,7 @@ class ComponentXType(Enum):
     SUPERGRID = "superGrid"
     PAGINATEDSUPERGRID = "paginatedSuperGrid"
     FILTEREDGRID = "filteredGrid"
+    DISPLAYFIELD = 'displayfield'
```
Comment (jberends) [inline pykechain/enums.py]: consistent use of "

File: pykechain/enums.py [hunk 3]
```diff
@@ -18,7 +18,7 @@ def options(cls):
     @classmethod
     def values(cls):
         """Provide a (sorted) list of values."""
-        return [value for (name, value) in cls.options()]
+        return [value for (value, name) in cls.options()]
```
Comment (jberends) [inline pykechain/enums.py]: this was the err you were talking about right?

File: pykechain/__init__.py [hunk 4]
```diff
@@ -5,4 +5,4 @@
 
 __all__ = (
     'Client', 'version'
-)
+)
```
Comment (jberends) [inline pykechain/__init__.py]: seems that file should end with newline is not in your prefs'

**Pull request merged**